runbotics/runbotics-orchestrator-ui/src/components/cron/getCurrentLocale.ts

```diff
@@ -0,0 +1,92 @@
+import { translate, isNamespaceLoaded } from 'src/hooks/useTranslations';
+import { DefaultLocale } from './types';
+
+export const getCurrentLocale = async (): Promise<DefaultLocale> => {
+    try{
```

Comment:
bad indentation + redundant try/catch

Reply:
I compared this with Navbar.sections.ts but I can't spot what's wrong with indentation. 

Removed try/catch ✔